cmd/zlint/main.go

```diff
@@ -62,32 +71,40 @@ func init() {
 }
 
 func main() {
+	// Build a registry of lints using the include/exclude lint name and source
+	// flags.
+	registry, err := setLints()
+	if err != nil {
+		log.Fatalf("unable to configure included/exclude lints: %v\n", err)
+	}
 
 	if listLintsJSON {
-		zlint.EncodeLintDescriptionsToJSON(os.Stdout)
+		registry.WriteJSON(os.Stdout)
 		return
 	}
 
 	if listLintsSchema {
-		names := make([]string, 0, len(lint.Lints))
-		for lintName := range lint.Lints {
-			names = append(names, lintName)
-		}
-		sort.Strings(names)
+		names := registry.Names()
```

Comment:
This is fine now, but we should probably drop public support for ZSchema within ZLint.

Reply:
@dadrian Honestly I'm not 100% clear on what ZSchema is :laughing: I assumed it was important to Censys/ZMap but if it isn't I'm happy to rip it out now. We're already breaking back compat.

Reply:
I went ahead and ripped it out in 2a7674f. Can always revert that commit if folks would prefer a more gradual deprecation.